src/common/fileUtils.js

```diff
@@ -110,3 +119,39 @@ export const fileLatestStatus = fileNode => {
 export const lengthLimit = (text, limit) => {
   return text.length > limit ? text.substring(0, limit) + '...' : text;
 };
+
+// Sort files by created date
+export const createDateSort = (fileList, sortDirection) => {
```

Comment:
Instead of taking and returning a list of values, maybe just return the function?
Something where we can do `[file1, file2, ...].sort(createDateSort)`